_posts/2016-03-17-persistent-neglectedness.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title:  "Persistently neglected causes"
+date:   2016-03-17 11:58:26
+author: Michael Peyton Jones
+categories:
+---
+
+A heuristic that we often use when assessing causes is that of *neglectedness*.
+If a cause is neglected, then that means that it is not receiving an amount of
+attention commensurate to it's seriousness. Given that most causes suffer from
+diminishing marginal returns on investment, if less has been invested then we
+should expect there to be especially cost-effective projects still available.
+
+But not all neglectedness is made equal. Some causes are only neglected
+*transiently*. That is, they aren't receiving
+much attention now, but it's clear that that will change in the relatively near
+future. An example is mobile services for the poor. This cause has been
+neglected for a long time because the infrastructure and the handset penetration
+were not there, but now that they are interest is picking up, and
+we should expect to see an increasing amount of resources directed there over
+the coming years. So it's likely that many potential projects in this
+area will happen anyway in the next couple of decades.
+
+If a cause is only transiently neglected then the impact of paying attention to
+it is lesser. If someone else is going to pay that attention eventually,
+then by doing it now you have simply brought that time forward.
+That can be a great thing - getting a malaria vaccine even a year earlier would
+be fantastic! But it's even better if we can address a cause that is
+*persistently* neglected. In that case you are bringing forward an event which
+might otherwise require an unusual combination of factors to occur, and
+therefore probably wouldn't happen for a long time.
+
+Why might a cause be persistently neglected? One reason can be that there are systematic
+obstacles to addressing it. For example, many good things can be done by for-profit companies,
+but it there are a number of problems that can systematically obstruct for-profit
+solutions, for example:
+
+- The beneficiaries of the project can't pay for it.
+- It's difficult to capture the value created by the project (the beneficiaries may not be the customers).
+- There is a more profitable way to deliver the product to the customers that
+  doesn't have as much social benefit.
+
+And almost any project can suffer from more general, but still systematic
+problems, such as:
+
+- The project requires unusual skills.
+- The project requires unusual beliefs ("you can build a cheaper rocket").
```

Comment:
explain the 'cheaper rocket' comment with a reference, and maybe reference Peter Thiel's idea of secrets from zero to one